lte/gateway/c/li_agent/main.cpp

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <lte/protos/mconfig/mconfigs.pb.h>
+#include <thread>
+
+#include "MagmaService.h"
+#include "MConfigLoader.h"
+#include "ServiceRegistrySingleton.h"
+
+#include "InterfaceMonitor.h"
+#include "PDUGenerator.h"
+#include "ProxyConnector.h"
+
+#define LIAGENTD "liagentd"
+#define LIAGENTD_VERSION "1.0"
+
+static magma::mconfig::LIAgentD get_default_mconfig() {
```

Comment:
Here and below repeatedly, static should not be necessary in this scope... I don't think. Unless there is something strange going on with the logging library.

Reply:
just trying to stay consistent here with other services, not quite sure why its static everywhere we can address that separately if needed https://github.com/magma/magma/search?q=get_default_mconfig